Skyline.py

```diff
@@ -39,24 +42,70 @@ class BuildingTuple:
     #   1) [int]  height: passes the height of the desired building
     #   2) [int]  x_coordinate: passes the x-coordinate of the desired building
     #
-    def __init__(self, height:int, x_coordinate:int):
+    def __init__(self, height: int, x_coordinate: int):
         self.height = height
         self.x_coordinate = x_coordinate
+
     #
     # Function Name: __str__
     # Formal Parameters: None
     # Return Value:str
     # Usage: prints out the height and x-coordinate of the building as a string
     #
-    def __str__(self)-> str:
+    def __str__(self) -> str:
         return f"({self.height},{self.x_coordinate})"
 
+
 #
 # Class Name: Skyline
 # Class Variables: None
 # Usage: stores static methods that are used to generate the skyline from a group of buildings
 #
 class Skyline:
+
+    # ALGORITHM PSEUDOCODE
+    #
+    # This is the pseudocode for the recursive Skyline Identifier algorithm. The implementation in Python can be found right below the pseudocode.
+    # See README.md for execution instructions and test data information.
+    # --------------------------------------------------
+    #
+    # recursiveSkylineFunction(List B of buildings)
+    #     If there is only 1 building in B
+    #         Return building
+    #     Else:
+    #         Divide buildings into left list L and right list R
```

Comment:
Divide buildings/2 into left list L and right list R

Reply:
I'll have this directly committed to main